Kitodo/src/main/java/org/kitodo/production/services/data/PropertyService.java

```diff
@@ -156,8 +115,8 @@ private QueryBuilder getQueryForType(String type) {
      *            of the searched property
      * @return list of JSON objects with properties
      */
-    public List<Map<String,Object>> findByTitle(String title, String type, boolean contains) throws DataException {
-        return findProperty(PropertyTypeField.TITLE.getKey(), title, type, contains);
+    public List<Property> findByTitle(String title, String type, boolean contains) {
+        return getByQuery("from Property as property where property.title='" + title + "'");
```

Comment:
Please use `getByQuery` with second parameter with list of to map values. If not this is introducing a possible SQL injection.